cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -67,3 +67,30 @@ def waffle_flags():
     flag_name='library_authoring_mfe',
     module_name=__name__,
 )
+
+
+# .. toggle_name: studio.pages_and_resources_mfe
+# .. toggle_implementation: CourseWaffleFlag
+# .. toggle_default: False
+# .. toggle_description: Waffle flag to link existing studio views to the new Pages and Resources experience.
+# .. toggle_use_cases: temporary, open_edx
+# .. toggle_creation_date: 2021-05-24
+# .. toggle_target_removal_date: 2021-12-31
+# .. toggle_warnings: Also set settings.COURSE_AUTHORING_MICROFRONTEND_URL.
+# .. toggle_tickets: None
+ENABLE_PAGES_AND_RESOURCES_MICROFRONTEND = CourseWaffleFlag(
+    waffle_namespace=waffle_flags(),
+    flag_name='pages_and_resources_mfe',
+    module_name=__name__,
+)
```

Comment:
?

Reply:
oh this seems to be from me rebasing my code from master...someone must have put a new flag in

Reply:
Looks like it was moved to https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/discussions/config/waffle.py#L38. You're probably safe just deleting those lines of code.